tasks/phantom-dc.rake

```diff
@@ -331,6 +331,20 @@ namespace :'phantom-dc' do
 
     puts "No congressional defaults found for the following members: " + notfound.inspect
   end
+
+  desc "Enable defunct status of congressmember"
+  task :defunct, :bioguide_id, :contact_url do |t, args|
+    cm = CongressMember.find_by!(bioguide_id: args[:bioguide_id])
+    attrs = { defunct: true }
+    attrs.merge!(contact_url: args[:contact_url]) if args[:contact_url]
+    cm.update!(attrs)
+  end
+
+  desc "Disable defunct status of congressmember"
```

Comment:
I assume contact_url exists to allow us to refer users to the official site of a defunct member of Congress, and lack of a contact_url means that we don't know where to send them, right?  If so, I think we want to delete the value when undefuncting a MOC, to avoid this mode of failure:
1. We discover an MOC is defunct, and there's a viable contact_url, so we run the defunct rake task with the contact URL.
2. The MOC can now be contacted, so we run undefunct.  The old contact_url is still in the DB.
3. Oh wait, the MOC is now defunct again, but with no viable contact_url anymore, so someone runs the defunct rake task with no contact_url.  Oops, the old contact_url is still in the DB.